### DISCUSSION TITLE
A quite feature riched wezterm configuration, just a show case for the new guys come from others terminal.

### DISCUSSION THREAD

**Discussion by zw963** [wezterm/wezterm#4952] [General]
Check my gist. [here](https://gist.github.com/zw963/b02e1cd320538a7276f0e0b666b2b8cc)

If you are a user from konsole, it support every thing from there, with more.

**Comment by miversen33** [comment]
Probably be more useful [here](https://github.com/wez/wezterm/discussions/628)

**Comment by zw963** [comment]
> Probably be more useful [here](https://github.com/wez/wezterm/discussions/628)

Thanks, i saw that post before, but, i consider those config there a bit of magic and advanced, my config more suitable for the newbie, most common feature.
